Output from /home/x_zhzhu/RoboSaGA/robomimic/robomimic/scripts/eval_trained_agent.py with arguments --agent ../experiments/robosaga/square_image/bc_rnn/saga/20240524021239/models/model_epoch_320_NutAssemblySquare_success_0.84.pth --n_rollouts 50 --distractors bottle lemon milk can --video_path ../experiments/robosaga/square_image/bc_rnn/saga/20240524021239/eval/videos/distractors_ckpt_model_epoch_320_NutAssemblySquare_success_0.84.mp4:

Output from /home/x_zhzhu/RoboSaGA/robomimic/robomimic/scripts/eval_trained_agent.py with arguments --agent ../experiments/robosaga/square_image/bc_rnn/saga/20240524021239/models/model_epoch_420.pth --n_rollouts 50 --distractors bottle lemon milk can --video_path ../experiments/robosaga/square_image/bc_rnn/saga/20240524021239/eval/videos/distractors_ckpt_model_epoch_420.mp4:

Output from /home/x_zhzhu/RoboSaGA/robomimic/robomimic/scripts/eval_trained_agent.py with arguments --agent ../experiments/robosaga/square_image/bc_rnn/saga/20240524021239/models/model_epoch_520.pth --n_rollouts 50 --distractors bottle lemon milk can --video_path ../experiments/robosaga/square_image/bc_rnn/saga/20240524021239/eval/videos/distractors_ckpt_model_epoch_520.mp4:

============= Initialized Observation Utils with Obs Spec =============

using obs modality: low_dim with keys: ['robot0_gripper_qpos', 'robot0_eef_quat', 'robot0_eef_pos']
using obs modality: rgb with keys: ['robot0_eye_in_hand_image', 'agentview_image']
using obs modality: depth with keys: []
using obs modality: scan with keys: []

Errors: Traceback (most recent call last):
  File "/home/x_zhzhu/RoboSaGA/robomimic/robomimic/scripts/eval_trained_agent.py", line 474, in <module>
    run_trained_agent(args)
  File "/home/x_zhzhu/RoboSaGA/robomimic/robomimic/scripts/eval_trained_agent.py", line 247, in run_trained_agent
    policy, ckpt_dict = FileUtils.policy_from_checkpoint(
  File "/home/x_zhzhu/RoboSaGA/robomimic/robomimic/utils/file_utils.py", line 346, in policy_from_checkpoint
    model = algo_factory(
  File "/home/x_zhzhu/RoboSaGA/robomimic/robomimic/algo/algo.py", line 74, in algo_factory
    return algo_cls(
  File "/home/x_zhzhu/RoboSaGA/robomimic/robomimic/algo/algo.py", line 121, in __init__
    self._create_networks()
  File "/home/x_zhzhu/RoboSaGA/robomimic/robomimic/algo/bc.py", line 593, in _create_networks
    self.nets = self.nets.float().to(self.device)
  File "/home/x_zhzhu/.conda/envs/robosaga/lib/python3.8/site-packages/torch/nn/modules/module.py", line 927, in to
    return self._apply(convert)
  File "/home/x_zhzhu/.conda/envs/robosaga/lib/python3.8/site-packages/torch/nn/modules/module.py", line 579, in _apply
    module._apply(fn)
  File "/home/x_zhzhu/.conda/envs/robosaga/lib/python3.8/site-packages/torch/nn/modules/module.py", line 579, in _apply
    module._apply(fn)
  File "/home/x_zhzhu/.conda/envs/robosaga/lib/python3.8/site-packages/torch/nn/modules/module.py", line 579, in _apply
    module._apply(fn)
  [Previous line repeated 1 more time]
  File "/home/x_zhzhu/.conda/envs/robosaga/lib/python3.8/site-packages/torch/nn/modules/rnn.py", line 189, in _apply
    self.flatten_parameters()
  File "/home/x_zhzhu/.conda/envs/robosaga/lib/python3.8/site-packages/torch/nn/modules/rnn.py", line 175, in flatten_parameters
    torch._cudnn_rnn_flatten_weight(
RuntimeError: cuDNN error: CUDNN_STATUS_NOT_INITIALIZED


===== Results for ../experiments/robosaga/square_image/bc_rnn/saga/20240524021239/ =====
Indomain: [0.84, 0.82, 0.82], 0.83 +/- 0.01
  data: [0.84, 0.82, 0.82]
Off-domain for distractors: nan +/- nan
   data: []
